modules/apps/marketplace/marketplace-app-manager-web/src/main/java/com/liferay/marketplace/app/manager/web/internal/util/AppDisplayFactoryUtil.java

```diff
@@ -243,7 +243,13 @@ protected static void filterAppDisplays(
 		while (iterator.hasNext()) {
 			AppDisplay appDisplay = iterator.next();
 
-			if ((state > 0) && (appDisplay.getState() != state)) {
+			List<Bundle> appDisplayBundles = appDisplay.getBundles();
+
+			int appDisplayBundlesSize = appDisplayBundles.size();
```

Comment:
No need to define this variable.  We can just check if `appDisplayBundles.size() == 0`.

Reply:
This one is not on me, the source formatter made me do it :)

Reply:
Fair enough!